### DISCUSSION TITLE
How to change jekyll-theme-cayman buttons?

### DISCUSSION THREAD

**Discussion by jackpittenger** [community/community#23209] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, I am currently making a project and I am wanting to use GitHub pages to host docs for it. I am using the jekyll-theme-cayman theme for it and I have my page pretty good so far :). The only problem is, I want to change/add this button. Is there any way to do this?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/0a115c8046db8e5976277242e3098f06d05b1c49.png" data-download-href="/uploads/short-url/1r3YdfaNCesswoJyygNGbc5YPZ7.png?dl=1" title="W4PCWEP2SwaA-J6Xc5DzqA" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/31329139/181075608-4c9e0e0b-22aa-411a-b175-578da9091926.png" alt="W4PCWEP2SwaA-J6Xc5DzqA" data-base62-sha1="1r3YdfaNCesswoJyygNGbc5YPZ7" width="690" height="224" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/0a115c8046db8e5976277242e3098f06d05b1c49_2_690x224.png, https://aws1.discourse-cdn.com/github/optimized/1X/0a115c8046db8e5976277242e3098f06d05b1c49_2_1035x336.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/0a115c8046db8e5976277242e3098f06d05b1c49.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/0a115c8046db8e5976277242e3098f06d05b1c49_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">W4PCWEP2SwaA-J6Xc5DzqA</span><span class="informations">1062×346 59.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by birdcar** [comment, marked as answer]
<p>Not sure if you’re still looking for an answer to your question here, but the way you’d change the Cayman theme’s CSS for the <code>.btn</code> class would be by following the instructions laid out here:</p>
<p><a href="https://help.github.com/articles/customizing-css-and-html-in-your-jekyll-theme/" rel="nofollow noopener">https://help.github.com/articles/customizing-css-and-html-in-your-jekyll-theme/</a></p>
<p>Hope that helps! </p>

**Comment by mianasbat** [comment]
<p>I was able to change the button colour, button border, button effects on hover.<br>
To do this first create a directory in <code>assets/css</code> and in root directory create a file <code>style.scss</code>. In that file paste the following code</p>
<pre><code class="lang-auto">---
---

@import "{{ site.theme }}";

  .btn {
    color: #0000ff;
    border-color: #0000ff; 
    // background-color: #008CBA;
  } 

  .btn:hover {
    color: #0bb6e1cc;
    text-decoration: none;
    background-color: #7c940ea1;
    border-color: #0000ff;
  }

</code></pre>
